doc/INSTALL.txt

```diff
@@ -128,107 +73,21 @@ cd odemis
 python ./setup.py build
 sudo ./setup.py install --root=/ --install-scripts=/usr/local/bin/ --install-data=/usr/ --single-version-externally-managed
 
-To get the fluorophores database, you can run this command:
+To get the fluorophores database (for the SECOM/DELPHI), you can run this command:
 ./util/getfluodb.py
 Then copy all the fluodb directory to /usr/share/odemis/fluodb/
 
 = Installing the vendor device drivers =
```

Comment:
The following paragraph now doesn't really say anything anymore about installing the vendor device drivers. Maybe add something like, see manufacturers for installation instructions.